src/examples/cancel/Shootist.java

```diff
@@ -56,18 +76,12 @@
 
     private boolean requestTerminated;
 
-    private static Logger logger = Logger.getLogger(Shootist.class);
-
-
-
-
-
     public void processRequest(RequestEvent requestReceivedEvent) {
         Request request = requestReceivedEvent.getRequest();
         ServerTransaction serverTransactionId = requestReceivedEvent
                 .getServerTransaction();
 
-        logger.info("\n\nRequest " + request.getMethod()
+        LOG.info("\n\nRequest " + request.getMethod()
```

Comment:
I agree there are conventions to set var names, but JAINSIP stack is an old guy, there are lots of style problem everywhere. I would perefer to modify as less files as possible, to make the commit targetted to log4j upgrade. If you are concerned about code convention, we can open a new issue, and cover code convention problems regarding log4j and more ( there are plenty of them).

My point on Log4jAPI was that I would be ok to modify 170 files for the log4j2 upgrade, if they really introduce an enhacement in terms of logging usage, but just changing the Logger var name is not reason enough to change those 170 files IMO.